lib/kitchen/directions/bake_chapter_review_exercises/v1.rb

```diff
@@ -32,7 +32,11 @@ def bake(chapter:, metadata_source:, append_to:, klass:)
 
       @content = exercise_clipboard.paste
 
-      append_to.append(child: render(file: 'review_exercises.xhtml.erb'))
+      append_to_element = append_to || chapter
+      @tag = append_to ? 'h3' : 'h2'
+      @metadata_title = append_to ? I18n.t(:eoc_exercises_title) : @title
```

Comment:
Even though Codecov says it's covered, I notice that the specs for v1 & v2 only have an example of this method being called with an append_to element, not without, so if there's anything unexpected with the behavior around appending to chapter, the spec won't catch it. Could you add that example to [the specs](https://github.com/openstax/kitchen/tree/main/spec/directions/move_exercises_to_eoc)?

Reply:
unless I'm missing something...

Reply:
Thank you for catching this :relaxed: I will try to add all instances to specs.